generators/app/templates/client/app/components/date-interceptor/date-interceptor.js

```diff
@@ -1,9 +1,10 @@
 (function () {
   'use strict';
+  var regexIso8601 = /^(\d{4}|\+\d{6})(?:-(\d{2})(?:-(\d{2})(?:T(\d{2}):(\d{2}):(\d{2})\.(\d{1,})(Z|([\-+])(\d{2}):(\d{2}))?)?)?)?$/;
```

Comment:
Nice code.
I tried to use moment, but I didn't come up with this way.
I thought `moment(input).isValid()` was not enough.